### PR TITLE
Operating Manual v0.2

### PR DESCRIPTION
**Pull request by ben-chain**
**This Pull Request is the Optimism Foundation's current draft for v0.2 of the Operating Manual.** For discussion and a changelog, please [visit the forum](TODOlink)!

### CODE REVIEW

File: manual.md [hunk 1]
```diff
@@ -118,4 +122,14 @@ The Optimism Foundation will undertake this ministerial work with a view towards
 
 ### **Amendments**
 
-The procedures described in this OPerating Manual will go into effect at the time of Airdrop #1, and be adjusted later in connection with a series of governance experiments. These changes include but are not limited to introducing the Citizens’ House, and adding, removing, and modifying Proposal Types and rules relating to voting processes. Any non-clerical updates to the OPerating Manual will be reflected with a new version number at the top of this document, at which point the updated version will go into effect.
+The procedures described in this Operating Manual will go into effect as releases are published on GitHub. Major releases to the manual will be made in connection with a series of governance experiments (“Seasons”). These changes include but are not limited to introducing the Citizens’ House, and adding, removing, and modifying Proposal Types and rules relating to voting processes. Any non-clerical updates to the Operating Manual will be reflected with a new version number at the top of this document, at which point the updated version will go into effect.
+
+## Process TLDR
```
Comment (timbeiko) [inline manual.md:127]: This should probably be at the top of the doc, with links to document sections that go into more depths. Maybe like a table of contents/TL;DR mix?

File: manual.md [hunk 2]
```diff
@@ -1,102 +1,106 @@
...
 
-In carrying out these functions, OP holders may either vote their OP directly, or delegate their OP voting power to an eligible third party.
+In carrying out these functions, OP holders may either vote their OP directly (by delegating their OP voting power to their own address), or delegate their OP voting power to an eligible third party. Addresses with delegated OP voting power are called “delegates.” Delegates are expected to exercise their authority responsibly and in accordance with community standards.
 
-The Token House is just the first part of the Optimism Collective [governance structure](https://community.optimism.io/docs/governance/). Launch of the Citizens’ House will soon follow and this OPerating Manual will be updated accordingly.
+The Token House is just the first part of the Optimism Collective [governance structure](https://community.optimism.io/docs/governance/). Launch of the Citizens’ House will soon follow and this Operating Manual will be updated accordingly.
 
 ### **Governance Toolkit**
 
 The two primary tools for Optimism Collective governance are:
```
Comment (timbeiko) [inline manual.md:15]:
```suggestion
The primary tools for Optimism Collective governance are:
```
list has 3 things

File: manual.md [hunk 3]
```diff
@@ -1,102 +1,106 @@
...
 
 - [**Snapshot**](https://snapshot.org/#/opcollective.eth): Where all Token House governance proposals are submitted for vote.
 - [**The Optimism Forum**](http://gov.optimism.io): A platform for discussion and deliberation about governance proposals.
+- [**Discord**](https://discord-gateway.optimism.io/): For informal governance discussion and feedback.
```
Comment (timbeiko) [inline manual.md:19]: might be worth listing the explicit channels to use?

File: manual.md [hunk 4]
```diff
@@ -1,102 +1,106 @@
...
+|Governance Fund  (Phase 1)      |OP distributions to proactively incentivize future growth of projects and communities in the Optimism ecosystem. Proposals should follow [this template](https://gov.optimism.io/t/governance-fund-phase-1-how-to-create-a-proposal/216).                                                             |Forum + Snapshot                                                                                                                                                                                                                                                                          |one three-week cycle                    | 10%                                                   |51%                                                                               |
```
Comment (timbeiko) [inline manual.md:90]:
```suggestion
|Governance Fund  (Phase 1)      |OP distributions to proactively incentivize future growth of projects and communities in the Optimism ecosystem. Proposals should follow [this template](https://gov.optimism.io/t/governance-fund-phase-1-how-to-create-a-proposal/216).                                                             |Forum + Snapshot                                                                                                                                                                                                                                                                          |three-week cycle                    | 10%                                                   |51%                                                                               |
```

would make this match the other rows for legibility

File: manual.md [hunk 5]
```diff
@@ -1,102 +1,106 @@
...
+|Protocol Upgrade                |Scheduled changes to the on-chain smart contracts comprising the mainnet Optimism protocol                                                                                                                                                                                                             |Snapshot                                                                                                                                                                                                                                                                                                                                                          |three-week cycle plus two week extended vote window                     | 10%                                                   | 51%                                                                               |
```
Comment (timbeiko) [inline manual.md:91]:
```suggestion
|Protocol Upgrade                |Scheduled changes to the on-chain smart contracts comprising the mainnet Optimism protocol                                                                                                                                                                                                             |Snapshot                                                                                                                                                                                                                                                                                                                                                          |three-week cycle **plus** two week extended vote window                     | 10%                                                   | 51%                                                                               |
```

would bold for emphasis, same on other lines

**Review by Gabo139 — APPROVED** [review]

**Pull request merged**